common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -114,7 +114,7 @@ def _create(cls, target_class, **kwargs):
         name = kwargs.get('name', kwargs.get('run', Location.clean(kwargs.get('display_name'))))
         run = kwargs.pop('run', name)
         user_id = kwargs.pop('user_id', ModuleStoreEnum.UserID.test)
-        emit_signals = kwargs.get('emit_signals', False)
+        emit_signals = kwargs.pop('emit_signals', False)
```

Comment:
Signals weren't being emitted and when I tried to pass in `emit_signals` in the documentation (like 105), I got an error.  I checked with @doctoryes about this change to the test factory.